ocaml/sdk-gen/csharp/autogen/src/Converters.cs

```diff
@@ -437,12 +439,16 @@ internal class XenDateTimeConverter : IsoDateTimeConverter
 
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
-            string str = JToken.Load(reader).ToString();
+            // JsonReader may have already parsed the date for us
```

Comment:
I remember that this had not been done on purpose because the default Json parsing did not always assume universal.